stable/bblfsh-web/templates/deployment.yaml

```diff
@@ -36,6 +36,20 @@ spec:
         - name: http
           containerPort: {{ .Values.service.internalPort }}
           protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /health
```

Comment:
Let's see how this works. There's no `/health` endpoint implemented as such but it should be better than a tcp probe